addons/website_livechat/static/tests/tours/website_livechat_chatbot_redirect.js

```diff
@@ -0,0 +1,50 @@
+/* @odoo-module */
+
+import tour from "web_tour.tour";
+
+tour.register(
+    "website_livechat.chatbot_redirect",
+    {
+        test: true,
+    },
+    [
+        {
+            trigger: ".o_livechat_button",
+        },
+        {
+            trigger: ".o_thread_message_content:contains(Hello, were do you want to go?)",
+        },
+        {
+            trigger: ".o_livechat_chatbot_stepAnswer:contains(Go to the #chatbot-redirect anchor)",
+        },
+        {
+            trigger: ".o_thread_message_content:contains(Tadam, we are on the page you asked for!)",
+            run() {
+                const url = new URL(location.href);
+                if (url.pathname !== "/" || url.hash !== "#chatbot-redirect") {
+                    throw new Error(
+                        "Chatbot should have redirected to the #chatbot-redirect anchor."
+                    );
+                }
+            },
+        },
+        {
+            trigger: ".o_livechat_chatbot_restart",
+        },
+        {
+            trigger: ".o_livechat_chatbot_stepAnswer:contains(Go to the /chabtot-redirect page)",
+        },
+        {
```

Comment:
You could help `prettier` a little bit, by replacing
```js
{
    test: true,
}
```
by `{ test: true }`

(yes, prettier accepts different format, and sometimes it can be dumb and chose more LOCs when it could be less)

Less LOCs for equivalent code eases reading whole test at a glance and better understand what is being tested. With so many LOCs I thought you copy-pasted steps by mistake